Source/ACE/Entity/Player.cs

```diff
@@ -456,6 +478,42 @@ public void Load(AceCharacter character)
             Session.Network.EnqueueSend(setTurbineChatChannels, general, trade, lfg, roleplay);
         }
 
+        public AceObject GetSavableCharacter()
+        {
```

Comment:
This creates a logical copy of the player object as an AceObject for saving.  Since saving will be done asynchronously we must create a logical copy of the player when we save, otherwise we're subject to race conditions on the player's updates (unless we do extensive locking, but that would be a poor choice).